Sources/ConfidenceProvider/ConfidenceFeatureProvider.swift

```diff
@@ -147,7 +149,9 @@ public class ConfidenceFeatureProvider: FeatureProvider {
         newContext: OpenFeature.EvaluationContext
     ) {
         if confidence == nil {
-            self.resolve(strategy: .fetchAndActivate, context: ConfidenceTypeMapper.from(ctx: newContext))
+            Task {
```

Comment:
I think we should be able to cancel the very first resolve as well.

```suggestion
            currentResolveTask = Task {
```

Reply:
we are going to remove the possibility of confidence object to be nil. but we added the fix for now.